### ISSUE TITLE
Windows version

### ISSUE THREAD

**Issue by xmha97** [ubuntu-flutter-community/musicpod#51]
Hi, Flutter can make cross platform application.
Please make the Windows version.
Thanks 🙏

**Comment by Feichtmeier** [comment]
hi @xmha97 
Flutter makes it possible to build an app for different platforms yes. But in practice this often means that you need to make changes for the different platforms which increases the work
If you are looking for a flutter music player for different platforms I highly recommend [Harmonoid](https://harmonoid.com/)

I fear if you specifically look for this trinity local audio/podcast/radio you need to use three apps instead : /

So TLDR this is not planned sorry :(
Thanks for your interest though and best regards
